packages/esm-service-queues-app/src/queue-table/transitions/transition-queue-entry-modal.component.tsx

```diff
@@ -121,10 +127,16 @@ const TransitionQueueEntryModal: React.FC<TransitionQueueEntryModalProps> = ({ q
                   invalidText="Required"
                   value={formState.selectedQueue}
                   onChange={(event) => setSelectedQueueUuid(event.target.value)}>
-                  {queues?.map((queue) => (
-                    <SelectItem key={queue.uuid} text={queue.display} value={queue.uuid}>
-                      {queue.display}
-                    </SelectItem>
+                  {queues?.map(({ uuid, display }) => (
+                    <SelectItem
+                      key={uuid}
+                      text={
+                        uuid == queueEntry.queue.uuid
+                          ? t('currentValueFormatted', '{{value}} (Current value)', { value: display })
```

Comment:
I would probably just say "Current", I don't love the use of the term "value".  Same applies to status and priority.